src/System.Configuration.ConfigurationManager/tests/System/Configuration/SettingElementTests.cs

```diff
@@ -0,0 +1,69 @@
+﻿namespace System.Configuration
+{
+
+    using Xunit;
+
+    public class SettingElementTests
+    {
+
+        [Fact]
+        public void TestForInequality()
+        {
+            SettingElement ElementOne = new SettingElement("NotEqualOne", SettingsSerializeAs.String);
+            SettingElement ElementTwo = new SettingElement("NotEqualTwo", SettingsSerializeAs.String);
+            Assert.False(ElementOne.Equals(ElementTwo));
+        }
+
+        [Fact]
+        public void TestForEquality()
+        {
+            SettingElement ElementOne = new SettingElement("TheExactSameName", SettingsSerializeAs.String);
+            SettingElement ElementTwo = new SettingElement("TheExactSameName", SettingsSerializeAs.String);
+            Assert.True(ElementOne.Equals(ElementTwo));
+        }
+
+        [Fact]
+        public void AssureDefaultSettingSerializationIsString()
+        {
+            SettingElement Element = new SettingElement();
+            Assert.Equal(Element.SerializeAs, SettingsSerializeAs.String);
+        }
+
+        [Fact]
+        public void AssureDefaultNameIsEmptyString()
```

Comment:
Nit, I think "Assure" should probably be "Ensure". Or, many people would just miss off the verb at the front.